metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Study.java

```diff
@@ -84,9 +86,15 @@
     @Column(nullable = false)
     private LocalDate releaseDate;
 
-    @ApiModelProperty(position = 7, dataType = "java.lang.String", notes = "Url to a Taxonomy")
+    @ApiModelProperty(position = 7, dataType = "java.lang.String", notes = "URL to a Taxonomy")
```

Comment:
`URL to taxonomy`. Class names start with Caps but message specifies it and not necessarily class name

Reply:
Good point, fixed